### ISSUE TITLE
quality of mp3

### ISSUE THREAD

**Issue by alexmamonchik** [TrevorHinesley/carrierwave-audio#5]
looks like we can't rewrite a quality of a final mp3 file and locked to 128 kbps. Is it interesting if I can create some PR with ability to set 320 kbps?

**Comment by TrevorHinesley** [comment]
You should be able to set the quality via the `compression` output argument. See `output_options` [here](https://github.com/TrevorHinesley/carrierwave-audio#convert).

**Comment by alexmamonchik** [comment]
sorry, here is my code:
```
class AudioUploader < CarrierWave::Uploader::Base
  include CarrierWave::Audio
  storage :file

  version :mp3 do
    process convert: [output_options: { compression: 320 }]
  end

  def full_filename(for_file)
    "#{super.chomp(File.extname(super))}.mp3"
  end

  def extension_whitelist
    %w(mp3 wav ogg)
  end
```
and it doesn't work

**Comment by alexmamonchik** [comment]
@TrevorHinesley please, look at #6

**Comment by TrevorHinesley** [comment]
That's perfect, sorry about that! I'll cut a new gem version.

**Comment by TrevorHinesley** [comment]
Released in [V 1.0.9](https://github.com/TrevorHinesley/carrierwave-audio/releases/tag/v1.0.9)
